docs/pages/includes/reference/resources/autoupdate_config.mdx

```diff
@@ -0,0 +1,63 @@
+```yaml
+kind: autoupdate_config
+metadata:
+  name: autoupdate-config
+spec:
+  agents:
+    # mode allows users to enable, disable, or suspend agent updates at the
+    # cluster level. Disable agent automatic updates only if self-managed
+    # updates are in place. This value may also be set in autoupdate_version.
+    # If set in both places, disabled overrides suspended, which overrides enabled.
+    # Possible values: "enabled", "disabled", "suspended"
+    # Default: "disabled" (unless specified in autoupdate_version)
+    mode: enabled
+
+    # strategy used to roll out updates to agents.
+    # The halt-on-error strategy ensures that groups earlier in the schedule are
+    # given the opportunity to update to the target_version before groups that are
+    # later in the schedule. (Currently, the schedule must be stopped manually by
+    # setting the mode to "suspended" or "disabled". In the future, errors will be
+    # detected automatically).
+    # The time-based strategy ensure that each group updates within a defined
+    # time window, with no dependence between groups.
+    # Possible values: "halt-on-error" or "time-based"
+    # Default: "halt-on-error"
+    strategy: halt-on-error
```

Comment:
Can you have multiple autoupdate configs to apply different strategies to different groups?